docs/tutorial.rst

```diff
@@ -413,7 +413,7 @@ Are you familiar with passing in `*args` and `**kwargs` in Python?::
   >>> args = [1 2]
   >>> kwargs = {"keyword2": 3
   ...           "keyword1": 4}
-  >>> optional_arg(*args, **kwargs)
+  >>> optional-arg(*args, **kwargs)
```

Comment:
Ah. Then I think it should be `optional_arg` everywhere since Hy supports it, no?